docs/cloud-tutorials/cloud-cli.md

```diff
@@ -199,7 +199,7 @@ It executes the migration commands declared in `dbos-config.yaml`, deploys the a
 **Parameters:**
 - `[application-name]`: The name of the application to deploy. By default we obtain the application name from `dbos-config.yaml`. This argument overrides the package name.
 - `-d, --database <string>`: The name of the Postgres database instance to which this application will connect. This may only be set the first time an application is deployed and cannot be changed afterwards.
-- `--enable-timetravel`: Enable time travel for this application. This may only be set the first time an application is deployed and cannot be changed afterwards.
+- `--enable-timetravel`: Enable experimental time travel for this application. This may only be set the first time an application is deployed and cannot be changed afterwards.
```

Comment:
The first time *AN* application is deployed or the first time *THIS* application is deployed?